docs/pages/foundations/tokens/colour.tsx

```diff
@@ -71,21 +72,22 @@ export default function TokensColorPage() {
 							/>
 						</div>
 					</Box>
-					<h3 id="foreground">Foreground colors</h3>
-					<p>
+					<h2 id="foreground-colours">Foreground colours</h2>
```

Comment:
I believe these should be H2s as we:

- Link to them from the in-page nav which is reserved for H2s.
- Think it makes sense from a content POV